lib/travis/api/v3/models/branch.rb

```diff
@@ -0,0 +1,8 @@
+module Travis::API::V3
+  class Models::Branch < Model
+    belongs_to :repository
+    belongs_to :last_build, class_name: 'Travis::API::V3::Models::Build'.freeze
+    has_many   :builds,  foreign_key: [:repository_id, :branch], primary_key: [:repository_id, :name], order: 'id DESC'.freeze
```

Comment:
Confirm, but as long as we don't have a use case for that (UI or
otherwise), I'm not sure it's needed.

On Tue, Feb 24, 2015 at 12:52 AM, Josh Kalderimis <notifications@github.com>
wrote:

> In lib/travis/api/v3/models/branch.rb
> <https://github.com/travis-ci/travis-api/pull/164#discussion_r25215555>:
>
> > @@ -0,0 +1,8 @@
> > +module Travis::API::V3
> > +  class Models::Branch < Model
> > +    belongs_to :repository
> > +    belongs_to :last_build, class_name: 'Travis::API::V3::Models::Build'.freeze
> > +    has_many   :builds,  foreign_key: [:repository_id, :branch], primary_key: [:repository_id, :name], order: 'id DESC'.freeze
>
> true! which is also nice to know that we can have a relation for PR builds
> too.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/travis-ci/travis-api/pull/164/files#r25215555>.
>